src/main/java/edu/usf/cutr/grha/gtfs/TncToGtfsWriter.java

```diff
@@ -0,0 +1,127 @@
+package edu.usf.cutr.grha.gtfs;
+
+import edu.usf.cutr.grha.model.ChicagoTncData;
+import org.onebusaway.gtfs.model.*;
+import org.onebusaway.gtfs.model.calendar.ServiceDate;
+import org.onebusaway.gtfs.serialization.GtfsWriter;
+
+import java.io.File;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class TncToGtfsWriter {
+
+    public String filePath;
+
+    public TncToGtfsWriter(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void write(List<ChicagoTncData> chicagoTncDataList) {
+        GtfsWriter writer = new GtfsWriter();
+        writer.setOutputLocation(new File(filePath));
+
+        // Creates a single fake agency in agency.txt for TNC trips
+        Agency agency = newFakeAgency();
+        writer.handleEntity(agency);
+
+        // Creates a single fake route in routes.txt that all TNC trips can be assigned to
+        Route route = newFakeRoute(agency);
+        writer.handleEntity(route);
+
+        int counter = 0;
+        for (ChicagoTncData tncData: chicagoTncDataList) {
+
+            // Creates a bus stop (for stops.txt) for the origin and destination location
+            Stop originStop = newStop(tncData, agency, counter, true);
+            counter++;
+
+            writer.handleEntity(originStop);
+
+            Stop destinationStop = newStop(tncData, agency, counter, false);
+            counter++;
+
+            writer.handleEntity(destinationStop);
+
+            // Create a record in calendar_dates.txt with a unique service_id
+            ServiceCalendarDate calendarDate = newCalenderDate(tncData, agency);
+            writer.handleEntity(calendarDate);
+
+            // Create a record to trips.txt
+            Trip trip = newTrip(tncData, route, calendarDate, agency);
+            writer.handleEntity(trip);
```

Comment:
You still need to write records to stop_times.txt

Reply:
Ah - I forgot to push it. My apologies!